### ISSUE TITLE
MelonDS Core, Futher Microphone Support?

### ISSUE THREAD

**Issue by k4adoo** [libretro/melonDS#101]
Are there any plans to add in further microphone support? Including using physical microphone or being able to use your own sample to make games like Bowser's Inside Story and Phantom Tracks work when you need specific amount of sound where the random noise buttons don't work?

**Comment by Sanaki** [comment]
From what I can see, being able to load a custom sample (supported in upstream, should be present in this codebase as well) and potentially having a core option to adjust the volume of the sample would cover over 90% of all microphone use cases on the system.

**Comment by k4adoo** [comment]
Yeah, I'm hoping we can get a fix for this, as it would easily fix most if not all microphone supported games that currently don't work.

**Comment by xunkar** [comment]
I'm really looking for an option to use an external microphone as input, like a real DS. This is necessary for a few games like Brain Age where you need to speak actual words. I'm also willing to put down a bounty for this feature, but how?

**Comment by tairosonloa** [comment]
I don't mind working on this, as this should be in the codebase already (from the fork). However, I would be grateful if someone could point me to some docs or how to / where to look to integrate it with the Retro Arch core menu or options.
